extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -411,6 +421,119 @@ function processCheckboxField(context: FieldContext): void {
 	context.onNewInputComponentCreated(context.fieldInfo.variableName!, checkbox);
 }
 
+// Values used for the Azure Account field inputs
+let selectedAccount: azdata.Account | undefined;
```

Comment:
i don't love this pattern inside of a utils class

Reply:
Yeah - that was my comment above about the specialized logic being here. I'm going to open an issue once I merge this in and follow up with @alanrenmsft on the design of this to make it better.